documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,53 +1,21 @@
 [id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Create a Prometheus `ServiceMonitor` that scrapes metrics from a {brandname} cluster.
+{ispn_operator} automatically creates a Prometheus `ServiceMonitor` that scrapes metrics from your {brandname} cluster.
 
-.Prerequisites
+.Procedure
 
-* Have an `oc` client.
 * Enable monitoring for user-defined projects on {ocp}.
 
-.Procedure
+When it detects that the `ServiceMonitor` resource is available on the {openshiftshort} cluster, {ispn_operator} adds the `infinispan.org/monitoring: 'true'` annotation to your `Infinispan` CR.
 
-. Create an authentication secret for the `operator` user.
-+
-Prometheus needs credentials to authenticate with {brandname} and must use the `operator` user and corresponding password.
-+
-.. Retrieve the `operator` user credentials.
-+
-[source,options="nowrap",subs=attributes+]
-----
-include::cmd_examples/oc_get_secret_operator_creds.adoc[]
-----
-+
-.. Create an authentication secret, for example:
-+
-[source,options="nowrap",subs=attributes+]
-----
-include::yaml/basic_auth.yaml[]
-----
-+
-.. Add the authentication secret to your {brandname} cluster namespace.
-+
-[source,options="nowrap",subs=attributes+]
-----
-$ oc apply -f basic-auth.yaml
-----
-+
-. Create a Prometheus `ServiceMonitor` that scrapes {brandname} metrics.
-+
-[source,options="nowrap",subs=attributes+]
+[source,yaml,options="nowrap",subs=attributes+]
 ----
-include::yaml/prometheus_service_monitor.yaml[]
-----
-+
-. Add the `ServiceMonitor`.
-+
-[source,options="nowrap",subs=attributes+]
-----
-$ oc apply -f service-monitor.yaml
+include::yaml/infinispan-monitoring-true.yaml[]
 ----
 
+{ispn_operator} also creates a `ServiceMonitor` named `<cluster_name>-monitor` and an authentication secret named `<cluster_name>-monitor-secret`.
```

Comment:
@oraNod please pay attention, that operator no longer creates a separate Secret and just utilizing single Secret for all functions

Reply:
fixed